src/components/NewGameButton.js

```diff
@@ -0,0 +1,22 @@
+const NewGameButton = (onButtonClick) => {
+  const button = document.createElement('button');
+  const firstLetter = document.createElement('firstletter');
+  const elementNum = document.createElement('span');
+
+  button.classList.add('newGameButton');
+  firstLetter.classList.add('newGameButton__first-letter');
+  elementNum.classList.add('elementNum');
+
+  button.textContent = 'ew Game';
+  firstLetter.textContent = 'N';
+  elementNum.textContent = '7';
```

Comment:
```suggestion
  const  text = 'New Game';
  button.textContent = 'tutaj jakaś funkcja';
  firstLetter.textContent = 'tutaj jakaś inna funkcja';
  elementNum.textContent = '7';
```
Spróbuj poszukać w necie czego użyć (za pomocą  JS), aby ze zmiennej "text" wyciągnąć pierwszą literę oraz resztę tekstu

Reply:
Dodane